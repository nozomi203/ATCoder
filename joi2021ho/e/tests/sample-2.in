10 10
1 8 9 8 1 5 7 10 6 6
10 10 2 8 10 3 9 8 3 7
2 11 28
5 11 28
7 11 28
1 11 18
3 11 18
8 11 18
4 11 11
6 11 11
10 11 11
9 11 5
