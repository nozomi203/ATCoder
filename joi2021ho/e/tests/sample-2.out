208
112
179
248
158
116
234
162
42
-1
